Tasks/app-store-release/app-store-release.ts

```diff
@@ -43,25 +43,17 @@ function findIpa(ipaPath: string): string {
     return paths[0];
 }
 
-async function isFastlaneInstalled(): Promise<boolean> {
+function isFastlaneInstalled(): boolean {
     tl.debug('Checking for previous fastlane version first...');
-    let gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
-    gemRunner.arg(['list', 'fastlane']);
-    gemRunner.arg(['-i']);
-    let fastlaneListOutput: string;
-
-    gemRunner.on('stdout', (data) => {
-        fastlaneListOutput = data.toString().trim();
-    });
-
-    await gemRunner.exec();
-
-    if (fastlaneListOutput === 'true') {
-        // fastlane is installed
-        return Promise.resolve(true);
-    } else {
-        return Promise.resolve(false);
+    const gemRunner: ToolRunner = tl.tool(tl.which('gem', true));
+    gemRunner.arg(['list', 'fastlane', '-i']);
+    const fastlaneListResult = gemRunner.execSync();
+    if (fastlaneListResult && 
```

Comment:
nit: I think it would be a bit cleaner to just return the boolean test.
```
return (fastlaneListResult && ...);
```

Reply:
Also, the extra space at the end of line 51, 52 are breaking the build.

Reply:
I thumbs-upped the comment (which appears to be irrevocable) but I think the condition is actually of type `string | boolean`